### PR TITLE
[Validator] Dutch translation fixes

### PR DESCRIPTION
**Pull request by ro0NL**
| Q | A |
| --- | --- |
| Branch? | "master" |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | ~ |
| License | MIT |
| Doc PR | ~ |

Some dutch translation fixes. I probably can rebase this against 2.*, but maybe someone wants to comment first or spots any other spelling errors.

Ignore fabbot.io :)

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 1]
```diff
@@ -4,7 +4,7 @@
         <body>
             <trans-unit id="1">
                 <source>This value should be false.</source>
-                <target>Deze waarde mag niet waar zijn.</target>
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Compared to english: `This value may not be true.`

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 2]
```diff
@@ -100,7 +100,7 @@
             </trans-unit>
             <trans-unit id="25">
                 <source>This value is not valid.</source>
-                <target>Deze waarde is ongeldig.</target>
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Compared to english: `This value is invalid.`

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 3]
```diff
@@ -140,15 +140,15 @@
             </trans-unit>
             <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Deze waarde representeert geen geldige taal.</target>
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Compared to english: `This value does not represents a valid language.`

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 4]
```diff
@@ -184,7 +184,7 @@
             </trans-unit>
             <trans-unit id="49">
                 <source>The file was only partially uploaded.</source>
-                <target>Het bestand is niet geheel geüpload.</target>
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Compared to english: `The file is not fully uploaded.`

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 5]
```diff
@@ -124,7 +124,7 @@
             </trans-unit>
             <trans-unit id="34">
                 <source>The file could not be uploaded.</source>
-                <target>Het bestand kon niet geüpload worden.</target>
+                <target>Het bestand kon niet worden geüpload.</target>
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Syntax

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 6]
```diff
@@ -56,7 +56,7 @@
             </trans-unit>
             <trans-unit id="14">
                 <source>The file could not be found.</source>
-                <target>Het bestand is niet gevonden.</target>
+                <target>Het bestand kon niet worden gevonden.</target>
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Syntax + past tense

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 7]
```diff
@@ -140,15 +140,15 @@
             </trans-unit>
             <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Deze waarde representeert geen geldige taal.</target>
+                <target>Deze waarde is geen geldige taal.</target>
             </trans-unit>
             <trans-unit id="39">
                 <source>This value is not a valid locale.</source>
-                <target>Deze waarde representeert geen geldige lokalisering.</target>
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: `lokalisering` = `localization`

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 8]
```diff
@@ -36,7 +36,7 @@
             </trans-unit>
             <trans-unit id="9">
                 <source>This field was not expected.</source>
-                <target>Dit veld was niet verwacht.</target>
+                <target>Dit veld werd niet verwacht.</target>
```
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: More common.

**Comment by xabbuh** [comment]
@WouterJ @Iltar Would you mind to have a look at this?

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf (hunk 4 above)
Comment (sstok) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: "Het bestand **is** slechts gedeeltelijk geüpload."

This is what gives the most results on Google.

**Comment by sstok** [comment]
Sorry to post this in Dutch, but else it becomes an unreadable mess :)

* `Moet` en `moeten` klinkt erg dwingend, `mag niet` klinkt wat vriendelijker.
* `kon niet gevonden worden` is meer gebruikelijk dan `kon niet worden gevonden.` Maar `Het bestand kon niet worden geüpload.` is dan weer duidelijker dan de vorige versie 👍

Vertalingen hoeven niet letterlijk te worden overgenomen, zolang als de oorspronkelijke bedoeling duidelijk is. Maar dat is slechts mijn mening, dus ik ben benieuwd wat andere vinden 😄

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 9]
```diff
@@ -140,15 +140,15 @@
             </trans-unit>
             <trans-unit id="38">
                 <source>This value is not a valid language.</source>
-                <target>Deze waarde representeert geen geldige taal.</target>
+                <target>Deze waarde is geen geldige taal.</target>
             </trans-unit>
             <trans-unit id="39">
                 <source>This value is not a valid locale.</source>
-                <target>Deze waarde representeert geen geldige lokalisering.</target>
+                <target>Deze waarde is geen geldige locale.</target>
```
Comment (iltar) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: If i throw this in google translate, I get "local" from Dutch "locale" to English. The dictionary doesn't disapprove it and recognizes it but has no explanation: http://www.woorden.org/woord/locale

However, the results are a odd, while it's technically correct, it feels wrong. I'd say locale is still better than it was before.

**Comment by iltar** [comment]
I agree with @sstok's suggestions. Other than that, looks good to me!

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf (hunk 4 above)
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: But that's present tense right, where english is written in past tense. :confused: im not a 
language purist though, nor is google ;-)

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf (hunk 9 above)
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Hmz i get correct results on google.nl, also vice versa.
image

`local` is something different (basicly refers something that belongs to a certain place, ie "the local food was great").

Anyway, `locale` definitely is the _right_ translation here.

**Comment by ro0NL** [comment]
@sstok 

> Moet en moeten klinkt erg dwingend, mag niet klinkt wat vriendelijker.

This is about `should` vs `must`, where `must` is used in dutch. However this counts for a lot of translations and is mixed and matched currently, ie
- `Deze waarde moet leeg zijn.` = `Deze waarde _mag niet_ leeg zijn` (OK)
- `This value should be null.` = `Deze waarde _moet_ null zijn.` (Reads as `This value _must_ be null.`)

Shall we change this right now?

> 'kon niet gevonden worden' is meer gebruikelijk dan 'kon niet worden gevonden'.

(Dutch) Zeker?, ik vind beide nu duidelijk (worden geupload en worden gevonden). Geen idee of hier grammatica regels bij komen kijken eigenlijk :sweat_smile: google translate vertaald het als "worden gevonden" in ieder geval, zonder verdere suggesties. :confused: 

> Vertalingen hoeven niet letterlijk te worden overgenomen, zolang als de oorspronkelijke bedoeling duidelijk is.

This is about `This value is not valid.` meaning the same as `This value is invalid`. Which both can separately translated in dutch. Sstok's point is it basically doesnt matter to much how you translate this (it means the same, ie `This value is invalid` could be translated as `... is ongeldig` (literal) or `... is not valid` (less literal, but the same thing). Currently i "corrected" this for `This value should be false.`, however i tend to agree with sstok (mostly pragmaticly) but still i think we should do a "correct" translation when possible (ie not changing syntax/words to much).

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf (hunk 9 above)
Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: It's not even in the "vandale" :| http://www.vandale.nl/opzoeken?pattern=locale&lang=nn

**Comment by ro0NL** [comment]
Ok sorry for the maybe incoherent story above. Rewrote it a bit earlier to clarify.

Anyway, for now translations with `should be`. `should not be` and `could not be` are at least consistent.
- `This value should be false.` was the only exception in dutch using different syntax for `This should be X`. Imho it still reads as well/friendly.
- `could not be` is the problem maybe, with my last commit at least the syntax is consistent. Imho all read as well, however @sstok got me confused as he states it depends on the past participle to determine the most common syntax. Dutch will understand both that's not the problem..

**Comment by ro0NL** [comment]
I revered 2 changes for `could not be`. I understand what @sstok means. In dutch one syntax sounds better/more fluent then the other depending on the verb (said it a few times ;-)). However i have no clear ref on this, but both are valid spelling.

Currently some translations still switch from past to present tense, i guess this is also somewhat context related eventually.

Long story short; this is good to go.

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 10]
```diff
@@ -100,7 +100,7 @@
             </trans-unit>
             <trans-unit id="25">
                 <source>This value is not valid.</source>
-                <target>Deze waarde is ongeldig.</target>
+                <target>Deze waarde is niet geldig.</target>
```
Comment (wouterj) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: What about "Dit is geen geldige waarde" ?

Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Eventually we can translate it in many ways, i chose for consistent syntax. Imho this reads as well/friendly/fluent given the context of these translations. I dont understand whats wrong with translating `not valid` to `niet geldig`.

Comment (ro0NL) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Point is i think these are good _default_ translations. If you want a certain tone, syntax, whatsoever you are probably using custom translations anyway (by using new translations keys or overriding the default ones).

**Comment by fabpot** [comment]
Thank you @ro0NL.

**Pull request closed without merging**